actions/utils.ts

```diff
@@ -328,3 +335,27 @@ export function sendSlack(message: string): boolean {
   });
   return true;
 }
+
+/**
+ * Attempts to verify that the contract at the given address implements the interface of the `ComposableCoW`
+ * contract. This is done by checking that the contract contains the selectors of the functions that are
+ * required to be implemented by the interface.
+ * 
+ * @remarks This is not a foolproof way of verifying that the contract implements the interface, but it is
+ * a good enough heuristic to filter out most of the contracts that do not implement the interface.
+ * 
+ * @dev The selectors are:
+ * - `cabinet(address,bytes32)`: `1c7662c8`
+ * - `getTradeableOrderWithSignature(address,(address,bytes32,bytes),bytes,bytes32[])`: `26e0a196`
+ * 
+ * @param code the contract's deployed bytecode as a hex string
+ * @returns A boolean indicating if the contract likely implements the interface
+ */
+export function isCompatible(code: string): boolean {
```

Comment:
nit: I would prefer to give a more specific name to this util. `isComposableCoWCompatible`

uber-nit: would it better to be more explicit here with the var name, sth like `composableCowBytecode`?

Reply:
Tracked in #13